docs/user-tests/user-test-01-template-anwender.md

```diff
@@ -0,0 +1,38 @@
+# User-Test für Anwender
```

Comment:
Da wir gerne den formellen Vorgaben zur geschlechterneutralen Sprache der TH folgen wollen, schlage ich vor: 
```suggestion
# User-Test für AnwenderInnen
```

Reply:
Können wir so machen mit den geschlechterneutralen Sprache, jedoch bin ich mir gerade nicht wirklich sicher, ob wir das in sämtlichen anderen Dokumenten auch schon so hatten. Wenn ich mich korrekt erinnere, dann hatten wir immer nur von Anwender und Entwickler geschrieben? 

Müssten wir dann mal schauen, ob wir das in allen Dokumenten dann auch hinterher einheitlich haben.